core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -482,7 +509,10 @@ public synchronized void gainPrimacy() {
     mAsyncJournalWriter
         .set(new AsyncJournalWriter(mRaftJournalWriter, () -> getJournalSinks(null)));
     mTransferLeaderAllowed.set(true);
-    LOG.info("Gained primacy.");
+
+    mLastFailoverDurationMs.set(System.currentTimeMillis() - getLastLeaderTime());
```

Comment:
I'm not sure what this measures...
AFAIU, last leader time is supposed to mark the last time this raft instance was a leader. And it might stay as non-leader for a long time...
So this subtraction yields something like; "the time this node was not a leader."
In that case I'm not sure why do we need such info.

Reply:
See response to [this comment](https://github.com/Alluxio/alluxio/pull/14572/files#r768952525).